openwisp-config/files/openwisp.agent

```diff
@@ -216,7 +216,7 @@ register() {
 	fi
 	# exit if response does not seem to come from openwisp controller
 	check_header $REGISTRATION_PARAMETERS
-	if [ ! is_http_status $REGISTRATION_PARAMETERS 201 ]; then
+	if ! [ is_http_status $REGISTRATION_PARAMETERS 201 ]; then
```

Comment:
With this change the registration is broken, the device keeps registering constantly. Please properly test your changes before opening pull requests.